### DISCUSSION TITLE
Custom grammar does not load

### DISCUSSION THREAD

**Discussion by TheOnlyTails** [vuejs/vitepress#4249] [Q&A] [answered]
I'm trying to load my custom grammar in as such:
```ts
import grammar from "../../syntax/nymph.tmLanguage.json" with { type: "json" };

export default defineConfig({
	markdown: {
		shikiSetup: async (shiki) => {
			await shiki.loadLanguage(grammar);
		}
	}
});
```
I've tried using `langauges: [grammar]` as well.

However, when opening the site, I get this message:
```
The language 'nym' is not loaded, falling back to 'txt' for syntax highlighting.
```
What's going on here?

My grammar is available [here](https://gist.github.com/TheOnlyTails/69af4f0c535cabf934af66a9beb45c6f).

**Comment by brc-dd** [comment, marked as answer]
Set it's name to `nym`. Alternatively, add this:

```ts
markdown: {
    languageAlias: {
      nym: 'Nymph'
    }
}
```

**Reply by TheOnlyTails** [reply]
That was it, thank you!
